srcs/job/job_launch.c

```diff
@@ -12,7 +12,7 @@ static int			s_fork_it(t_sh *sh, t_job *j, t_proc *p)
 
 	if ((p->envp = ft_array_dup(sh->envp)) == NULL)
 		return (ST_MALLOC);
-	if ((ret = env_update_from_cmd_line(&p->argv, &p->argc, &p->envp)) != ST_OK)
+	if ((ret = env_or_var_update_from_cmd_line(&p, &sh)) != ST_OK)
```

Comment:
here again, no need to have a pointer to a pointer to t_proc and t_sh structures :-)